osx/post_installation.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+
+# Global Variables
+SCRIPTPATH=$( cd $(dirname $0) ; pwd -P )
+
+KALITE_SHARED="/Users/Shared/ka-lite"
+KALITE_DIR="~/.kalite/"
+PYRUN_NAME="pyrun-2.7"
+PYRUN_DIR="$KALITE_SHARED/$PYRUN_NAME"
+PYRUN="$PYRUN_DIR/bin/pyrun"
+PYRUN_PIP="$PYRUN_DIR/bin/pip"
+BIN_PATH="$PYRUN_DIR/bin"
+ASSESSMENT_SRC="$KALITE_SHARED/assessment/assessment.zip"
+SHEBANGCHECK_PATH="$KALITE_SHARED/scripts/"
+
+SYMLINK_FILE="$KALITE_SHARED/pyrun-2.7/bin/kalite"
+SYMLINK_TO="/usr/local/bin"
+COMMAND_SYMLINK="ln -s $SYMLINK_FILE $SYMLINK_TO"
+
+TMP="/tmp/"
+ORG="org.learningequality.kalite"
+LAUNCH_AGENTS="/Library/LaunchAgents/"
+LAUNCH_DAEMONS="/Library/LaunchDaemons/"
+KALITE=$(which kalite)
+PLIST_SRC="$TMP$ORG.plist"
+PLIST_DST"$LAUNCH_DAEMONS$ORG.plist"
```

Comment:
Don't use LaunchDaemons on our .plist, it will not work because it's used for daemon services.  See: https://developer.apple.com/library/mac/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLaunchdJobs.html